docs/sphinx/using/examples/introduction.rst

```diff
@@ -0,0 +1,42 @@
+Introduction
+--------------------------------
+
+Welcome to CUDA Quantum! On this page we will illustrate CUDA Quantum with several examples. 
+
+.. tab:: Python
+
+   We're going to take a look at how to construct quantum programs through CUDA Quantum's `Kernel` API.
+
+   When you create a `Kernel` and invoke its methods, a quantum program is constructed that can then be executed by calling, for example, `cudaq::sample`. Let's take a closer look!
+
+   .. literalinclude:: ../../examples/python/intro.py
+      :language: python
+
+.. tab:: C++
+
+   We're going to take a look at how to construct quantum programs using CUDA Quantum kernel expressions.
```

Comment:
```suggestion
   We're going to take a look at how to construct quantum programs using CUDA Quantum kernels.
```
quantum kernels are functions, and hence calling them expressions seems odd from a language perspective.